src/libs/ReactNativeLauncher.js

```diff
@@ -55,11 +56,13 @@ class ReactNativeLauncher extends Launcher {
       'sendToPilot',
       'getCookiesByDomain',
       'getCookieByDomainAndName',
-      'getCookieFromKeychainByName'
+      'getCookieFromKeychainByName',
+      'saveCookieToKeychain'
```

Comment:
How is it related to this PR?

Reply:
Ho, this is from a another test. I will remove it, thanks